packages/core/admin/admin/src/pages/Settings/pages/Webhooks/components/HeadersInput.tsx

```diff
@@ -81,8 +91,7 @@ const HeadersInput = () => {
                       padding="0.4rem"
                       alignItems="center"
                       justifyContent="center"
-                      disabled={value.length === 1}
-                      onClick={() => removeFieldRow('headers', index)}
+                      onClick={() => removeRow(index)}
```

Comment:
The styling is a bit strange for the minus button. If we replace this second <Grid.Item> with this it looks better

```
              <Grid.Item col={6} direction="column" alignItems="stretch">
                <Flex alignItems="flex-end" gap={2}>
                  <Box style={{ flex: 1 }}>
                    <StringInput
                      name={`headers.${index}.value`}
                      aria-label={`row ${index + 1} value`}
                      label={formatMessage({
                        id: 'Settings.webhooks.value',
                        defaultMessage: 'Value',
                      })}
                      type="string"
                    />
                  </Box>
                  <IconButton
                    width="3rem"
                    height="4rem"
                    onClick={() => removeRow(index)}
                    color="primary600"
                    label={formatMessage(
                      {
                        id: 'Settings.webhooks.headers.remove',
                        defaultMessage: 'Remove header row {number}',
                      },
                      { number: index + 1 }
                    )}
                    type="button"
                  >
                    <Minus width="0.8rem" />
                  </IconButton>
                </Flex>
              </Grid.Item>
```

Reply:
I have a hard time know what change you are suggesting :D